src/config.ts

```diff
@@ -36,3 +37,11 @@ export class TransactionLimits {
 
   oldEnoughForWithdrawalLimit = () => this.config.oldEnoughForWithdrawal / MS_IN_HOUR
 }
+
+export const getSpecterWalletConfig = (config): SpecterWalletConfig => ({
```

Comment:
@bodymindarts really not sure if we must pass here "config" (yaml), if we have this function in the same file really not sure about needing 2 references. We can remove this method and do a cast (if this is just a parsing method) or just use this method (much better) without the param or at least as optional (if have no value we can load the default configuration file)

Reply:
You are right we don’t need the function argument. We can treat yamlConfig as in scope within this file unless there is a clear need somewhere. 

I don’t like the idea of a cast as we want to have a consistent way of loading defaults or flattening structure or handling other config related logic so we’ve decided a function is the way to go.

Reply:
Good catch, I had noticed that `yamlConfig` was local as well but I was thinking to maintain the pattern we setup just above in `TransactionLimits` to pass the config explicitly.

I think you're right though that if we're moving all these things into the `config.ts` module then we can remove that redundancy. For now, since nothing needs a different config I think I'll remove the parameter entirely, and if we need it for anything later on we can also add it back as an optional param which shouldn't affect any existing code.